src/Mutator/Boolean/TrueValue.php

```diff
@@ -54,12 +60,29 @@ public function mutate(Node $node)
         return new Node\Expr\ConstFetch(new Node\Name('false'));
     }
 
-    protected function mutatesNode(Node $node): bool
+    protected function mutatesNode(Node $node, array $mutatorSettings): bool
     {
         if (!($node instanceof Node\Expr\ConstFetch)) {
             return false;
         }
 
-        return $node->name->toLowerString() === 'true';
+        if ($node->name->toLowerString() !== 'true') {
+            return false;
+        }
+
+        $parentNode = $node->getAttribute(ParentConnectorVisitor::PARENT_KEY);
+        $grandParentNode = $parentNode !== null ? $parentNode->getAttribute(ParentConnectorVisitor::PARENT_KEY) : null;
+
+        if ($grandParentNode instanceof Node\Expr\FuncCall) {
+            $resultSettings = array_merge(self::DEFAULT_SETTINGS, $mutatorSettings);
+
+            $functionName = $grandParentNode->name->toLowerString();
```

Comment:
This will crash if a function with a variable name is used, e.g

```php
<?php

$a = 'foo';
$a(true);

'function_name'(true) || ('function_name')(true);
```

So we should add an `instanceof Node\Name` check before that part, and add tests with these cases.

Reply:
thanks, done